README.md

```diff
@@ -12,6 +12,9 @@ Example:
 ```
 ## Requires:
   * **python3**
+  * **python3-devel.x86_64**
```

Comment:
I assume that the `python-devel` package is needed to compile the PyInquirer`. If it is the case, you don't have to mention it separately.
Next, all dependencies beside `lxml` are part of the Python standard library, so you don't have to mention them as well.
The web browser is a non-Python dependency that almost everybody has, so you can express it differently (e.g. "The output of this utility is visualized in a web browser").